pkg/suites/suites.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package suites
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/networkservicemesh/cloudtest/pkg/model"
+)
+
+const noneParseFlag = 0
+
+func Find(root string) ([]*model.Suite, error) {
+	var testPaths []string
```

Comment:
@denis-tingajkin I've could not find a platform build filter check and I suppose we need it :-)

Reply:
@haiodo Note: we are adding suites only if the entry point of the suite is included into all found tests by cloudtest https://github.com/networkservicemesh/cloudtest/pull/38/files#diff-63ebc63ca05a2de0515117931dbf227dR1502

Reply:
@haiodo probably we need to add possible to search all source files by cloudtest (means do not call console golang command to search tests)
In this case, we'll duplicate logic with including/excluding test files by tags, but probably it makes cloudtest more flexible and we'll use only one project file walk(instead of 2 as it proposing in the current version).
What do you think?